src/com/shmuelzon/HomeAssistantFloorPlan/Controller.java

```diff
@@ -492,18 +520,18 @@ private Point2d getFurniture2dLocation(HomePieceOfFurniture piece) {
         return new Point2d((objectPosition.x * 0.5 + 0.5) * renderWidth, (objectPosition.y * 0.5 + 0.5) * renderHeight);
     }
 
-    private String generateStateIconYaml(String name, Point2d position, String type, String action) {
+    private String generateStateIconYaml(String name, Point2d position, String type, String action, String title) {
         String yaml = String.format(Locale.US,
             "  - type: state-%s\n" +
             "    entity: %s\n" +
-            "    title: null\n" +
+            "    title: %s\n" +
```

Comment:
If the description is empty, what will appear here (and in the HA frontend)? Will the behavior remain the same?